modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ViewObjectDefinitions/ViewObjectDefinitions.tsx

```diff
@@ -187,6 +191,31 @@ export default function ViewObjectDefinitions({
 		return url;
 	};
 
+	useEffect(() => {
+		if (objectFoldersRequestInfo?.items.length > 1) {
+			const itemsActions = [...objectDefinitionsFDSActionDropdownItems];
+			itemsActions.push({
+				data: {
+					id: 'moveObjectDefinition',
+					method: 'update',
+					permissionKey: 'update',
+				},
+				href: null,
+				icon: 'move-folder',
+				label: 'Move',
+				target: null,
+				type: 'item',
+			});
+			setUpdatedFDSItemsActions(itemsActions);
+		}
+		else if (objectFoldersRequestInfo?.items.length === 1) {
+			setUpdatedFDSItemsActions(objectDefinitionsFDSActionDropdownItems);
+		}
+	}, [
+		objectFoldersRequestInfo?.items.length,
+		objectDefinitionsFDSActionDropdownItems,
```

Comment:
`objectDefinitionsFDSActionDropdownItems` never changes, so it should not be here. Did you add this to satisfy eslint? I'd rather add a `eslint-disable-next-line react-hooks/exhaustive-deps` line.